bookkeeper-server/src/main/java/org/apache/bookkeeper/util/BookKeeperConstants.java

```diff
@@ -31,6 +31,7 @@
     public static final String BOOKIE_STATUS_FILENAME = "BOOKIE_STATUS";
     public static final String PASSWD = "passwd";
     public static final String CURRENT_DIR = "current";
+    public static final String METADATA_DIR = "metadata";

```

Comment:
I would suggest calling this a `entrylog_index_cache` directory. `metadata` is too ambitious and it indicates that it is a directory for storing persistent information.